src/RobotHub.cpp

```diff
@@ -76,7 +78,8 @@ void RobotHub::processRobotCommand(roboteam_proto::RobotCommand & cmd) {
     if (mode == utils::Mode::SERIAL) {
         sendSerialCommand(llrc);
     } else {
-        sendGrSimCommand(cmd);
+        //sendGrSimCommand(cmd);
```

Comment:
Maybe just as well remove GrSim support at all? Or maybe make something nice to switch between versions. Or send to both?

Reply:
I think just adding an additional mode to settings would be best, actually.